src/index.js

```diff
@@ -58,6 +59,16 @@ var handlers = {
 
     'GetReviewIntent': getReview,
 
+    'GetLatestReviewsIntent': getLatestReviews,
+
+    'ReviewTypeIntent': function() {
```

Comment:
Can we add a comment explaining that this is (presumably) invoked after a `getLatestReviews` with missing review type? Otherwise it's not clear what this is unless you know it (and read the code for `getLatestReviews`).